examples/nokia/srlinux-services/srl-openconfig.cfg.json

```diff
@@ -2361,9 +2401,6 @@
     }
   },
   "srl_nokia-network-instance:network-instance": [
-    {
-      "name": "DEFAULT"
-    },
```

Comment:
I would like this line to be reverted, helps align this topo with the featureprofiles defaults

Reply:
done [a5ecb13](https://github.com/openconfig/kne/pull/326/commits/a5ecb13dc037ad71e265feaa5e014e48499c83a2)